templates/web/base/report/_item_expandable.html

```diff
@@ -0,0 +1,55 @@
+[%#
+  This list item contains a full photo set and description, which is hidden
+  until the `.expanded` class is added. It is useful for times when you want
+  to surface the full information about a report, without requiring the user
+  to visit the dedicated report page.
+
+  Since the photo set includes `<a>` elements of its own, we drop the usual
+  "wrapper" `<a>` and the associated `.item-list--reports__item` class, to
+  avoid anchor nesting.
+%]
+
+[% truncated_detail = BLOCK %][% problem.detail | truncate(75, '…') | html_para %][% END ~%]
+[% full_detail = BLOCK %][% problem.detail | add_links | html_para %][% END ~%]
+[% SET show_more = truncated_detail != full_detail OR problem.photo ~%]
+
+<li class="[% 'js-toggle-expansion ' IF show_more %]item-list__item item-list__item--expandable [% item_extra_class %]"
+    data-report-id="[% problem.id | html %]"
+    id="report-[% problem.id | html %]">
+
+  [% IF problem.photo %]
+    <a href="[% c.cobrand.base_url_for_report( problem ) %][% problem.url %]" class="item-list__item--expandable__hide-when-expanded">
+        <img class="img" height="60" width="90" src="[% problem.photos.first.url_fp %]" alt="">
+    </a>
+  [% END %]
+
+  [% TRY %]
+    [% PROCESS 'report/_item_heading.html' %]
+  [% CATCH file %]
+    <h3>
+        <a href="[% c.cobrand.base_url_for_report( problem ) %][% problem.url %]">
+            [% problem.title | html %]
+        </a>
+    </h3>
+  [% END %]
+
+    <small>[% PROCESS 'report/_item_small.html' %]</small>
+
+    <p class="item-list__item--expandable__hide-when-expanded">
+      [% problem.detail | truncate(75, '…') | html %]
```

Comment:
@dracos Shame we can’t use `[% truncated_detail %]` here for neatness. Maybe we could, if we wrapped it in a `<div class="item-lis…panded">` instead of a `<p>`?

Reply:
Yeah, if that works for you, I wasn't sure if that could happen or not, go for it if it's okay.

Reply:
Yup, turns out it works fine, so I’ve made that change :+1: